tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -101,8 +103,14 @@ class CloudGov(Common):
     AWS_S3_STATICFILES_BUCKET_NAME = s3_staticfiles_creds['bucket']
     AWS_S3_STATICFILES_ENDPOINT = f'https://{s3_staticfiles_creds["endpoint"]}'
     AWS_S3_STATICFILES_REGION_NAME = s3_staticfiles_creds['region']
+
+    # TODO This was supose to be able to handle our s3 prefixes in 1 line, but
+    # we rolled a custom solution to multiple buckets with boto3 django extension
+    # instead of prefixing then. We'd like to be able to use this, as it is simpler
+    # AWS_LOCATION = cloudgov_name
+
     MEDIA_URL = \
-        f'{AWS_S3_STATICFILES_ENDPOINT}/{AWS_S3_STATICFILES_BUCKET_NAME}/'
+        f'{AWS_S3_STATICFILES_ENDPOINT}/{AWS_S3_STATICFILES_BUCKET_NAME}/{APP_NAME}/'
```

Comment:
This was the cause of my problem, at some point I must have removed the slash and it got commited with other changes